app/src/main/res/layout/item_image_preview_overlay.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We could get away with a FrameLayout here right? Just thing the performance might be a bit better

Reply:
The placement tags (ie layout_constraintEnd_toEndOf) at least require a ConstraintLayout.